### ISSUE TITLE
User icon and name is covering the bookmark icon in md-theme

### ISSUE THREAD

**Issue by SysUserR** [Dolibarr/dolibarr#29028]
### Bug

We are using the md-theme.
Since the upgrade to 19.0.1 the user icon and name is covering the bookmark-star-icon.
So the bookmarks are no more selectable. When selecting the bookmark icon with a mouse click the user data is shown (because it is overlaying the bookmark icon).

![Bild1](https://github.com/Dolibarr/dolibarr/assets/164546990/8e04a801-b001-4dda-86fd-6b28741b6da9)

 

### Dolibarr Version

19.0.1 running on WIN11

### Environment PHP

7.3.33

### Environment Database

mariadb 10.6.5

**Comment by ksar-ksar** [comment]
Hello,

I can't reproduce it on my V19.0.1 : 
![image](https://github.com/Dolibarr/dolibarr/assets/35605507/d95be360-5fa9-4bd3-a1ee-24688b371e20)

**Comment by SysUserR** [comment]
Hi,
your screen is much more wider than mine. My screen has 1920x1080 only.
![dolibarr-screenshot-240327](https://github.com/Dolibarr/dolibarr/assets/164546990/3677a6f1-9b23-4139-975b-afa59bb97877)

With version 18.0.5 I had the Boomark-icon left of the User-icon (like in your screenshot).
The position of the icon is not the real problem. Problem is the name displayed on the right side of the User-icon (the name is covering the Bookmark-icon).
I checked the Dolibarr demo website (with version 19.0.0). There the User-icon is displayed without the name.

I tried to change my user name in the settings. Removing the last name in the settings is not possible. When removing the prename in the settings then the last name is displayed (and covering the Bookmark-icon too).

As a workaround it would help when my prename is not displayed on the right side of the User-icon. Then the Bookmark-icon should be reachable again.

Regards

**Comment by SysUserR** [comment]
I have NO picture in my user settings.
I have now added a picture in my user settings and the problem is solved.
The User-icon is on the right side again, my prename is no more displayed.
![dolibarr-failure-solved-240327](https://github.com/Dolibarr/dolibarr/assets/164546990/ca0bd6d3-80ca-4d76-8ea8-a6a88266e7a0)

So it is a problem when you have no picture in the user settings.

**Comment by eldy** [comment]
If someone has the trouble, can you try this:
Clean your browser cache:  You can run CTRL + F5 on chrome on linux or windows for example (F5 is not enough)
This should be enough. If not, can you provide OS, browser version and screen resolution.

**Comment by eldy** [comment]
> I have NO picture in my user settings. I have now added a picture in my user settings and the problem is solved. The User-icon is on the right side again, my prename is no more displayed. ![dolibarr-failure-solved-240327](https://private-user-images.githubusercontent.com/164546990/317786965-ca0bd6d3-80ca-4d76-8ea8-a6a88266e7a0.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI0ODY2NzMsIm5iZiI6MTcxMjQ4NjM3MywicGF0aCI6Ii8xNjQ1NDY5OTAvMzE3Nzg2OTY1LWNhMGJkNmQzLTgwY2EtNGQ3Ni04ZWE4LWE2YTg4MjY2ZTdhMC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwN1QxMDM5MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMzBiMDg1MjViNjYzNzNjNjc1NDBiNWFlZGIwYmNlYTMyYTYyMTc3ZjNiMzljZTFlZTFiNDU5YWU3OWMxMzEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7VSNC-ND0fb2Cot7D0DsZeh9cN8cYBUXG9y6gWA91Pg)
> 
> So it is a problem when you have no picture in the user settings.

Not sure. Uploading an image may force the refresh of the CSS style file. So trouble may be just in the cache of your browser. What happen if you remove the picture now the problem is solved ?

**Comment by SysUserR** [comment]
Hi,

I tried this already.
When I delete the picture for the user I have the same problem like at the beginning again.
![dolibarr-bild-240407](https://github.com/Dolibarr/dolibarr/assets/164546990/fda16606-ee89-4c60-b4cc-2f6a3eef257f)

Procedure:
Picture is setup in the user settings.
Enter the user settings, delete the picture. Confirm with "Ändern" (in german).
Then there is NO change (depending on this problem) in the screen.
When I press F5 (refresh of the screen) then the user name and icon is covering the bookmark icon again.
So F5 makes the refresh.
I tried clearing of the cache, restart of the browser and reboot too, this does not help.

Setup:
Windows 11
Screen resolution 1920x1080px
Google Chrome, Version 123.0.6312.106 (Offizieller Build) (64-Bit), actual version
